### DISCUSSION TITLE
Entry Listing Column Sets

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#2971] [Feature Requests]
In a similar way to how "Publish Layouts" work with Channels when editing Entries, it would be great if we could create "Entry Listing Layouts" for when listing Entries from a selected Channel.

It would be sensible if we could pre-define which custom fields and standard fields (like Status, Date, Author) are displayed in the columns, when listing all the Entries from a selected Channel.
It would also make sense - just like with Publish Layouts - that these Column Sets could be applied on a per Member Role basis.

I can see there is some functionality along these lines in the Entry Listing Filter tools (EE6)... but I think this could be expanded.
Plus, for some reason, I'm not able to include my CartThrob Order Total field (cartthrob price - simple) as a Column.

[ Y ] I am capable and would like to work on implementation of this feature if it is considered.
Happy to help discuss/consult on this topic and help with design/testing.

**Comment by robinsowell** [comment]
I think re CartThrob, you can't currently make anything from Tabs a column.  It's only custom fields.  But we have that as a feature request (internally).

I was thinking what might also be nice- to me, if you include it as a column?  I'd ALSO like it to show up as a filter.  If you care enough about the data to want to show it in a column, you'd probably find it useful to be able to filter on it.

Would be really nice- text/textarea types, the filter would be a column specific search; a date field would get the same type filter as entry_date does; 

Lot of ways this could go, but definitely room to add some functionality.

**Comment by ic360-com** [comment]
Yes, it would be nice to be able to add any Field (where possible) into the Entry Listing columns. And then additionally provide a Filter for every column.
I used to like how Magento provided that feature - when I was working with Magento - for every column filterable.
More complex to set up initially... but powerful for the user.  :-)

![attribute-search](https://user-images.githubusercontent.com/36453035/186136865-52125909-bbba-4e78-a8b4-ffa09f0b4f83.png)
![Magento Column Selector](https://user-images.githubusercontent.com/36453035/186136900-b4c22271-d072-46f6-ace4-55119b1eba16.gif)
